File: src/main/java/arvem/skykid/CapeMaterial.java
package arvem.skykid;

import net.minecraft.item.ArmorItem;
import net.minecraft.item.ArmorMaterial;
import net.minecraft.recipe.Ingredient;
import net.minecraft.sound.SoundEvent;
import net.minecraft.sound.SoundEvents;

public class CapeMaterial implements ArmorMaterial {
    private static final int[] BASE_DURABILITY = new int[] {0, 0, 0, 0};
    private static final int[] PROTECTION_VALUES = new int[] {0, 0, 0, 0};

    @Override
    public int getDurability(ArmorItem.Type type) {
        return 0;
    }

    @Override
    public int getProtection(ArmorItem.Type type) {
        return 0;
    }

    @Override
    public int getEnchantability() {
        return 0;
    }

    @Override
    public SoundEvent getEquipSound() {
        return SoundEvents.ITEM_ARMOR_EQUIP_ELYTRA;
    }

    @Override
    public Ingredient getRepairIngredient() {
        return null;
    }

    @Override
    public String getName() {
        return "cape";
    }

    @Override
    public float getToughness() {
        return 0f;
    }

    @Override
    public float getKnockbackResistance() {
        return 0f;
    }

}

File: src/main/kotlin/arvem/skykid/registries/CapeMaterial.kt
package arvem.skykid.registries

import net.minecraft.item.ArmorItem
import net.minecraft.item.ArmorMaterial
import net.minecraft.recipe.Ingredient
import net.minecraft.sound.SoundEvent
import net.minecraft.sound.SoundEvents

class CapeMaterial : ArmorMaterial {
    override fun getDurability(type: ArmorItem.Type): Int {
        return 0
    }

    override fun getProtection(type: ArmorItem.Type): Int {
        return 0
    }

    override fun getEnchantability(): Int {
        return 0
    }

    override fun getEquipSound(): SoundEvent {
        return SoundEvents.ITEM_ARMOR_EQUIP_ELYTRA
    }

    override fun getRepairIngredient(): Ingredient {
        return Ingredient.EMPTY
    }

    override fun getName(): String {
        return "cape"
    }

    override fun getToughness(): Float {
        return 0f
    }

    override fun getKnockbackResistance(): Float {
        return 0f
    }
}